core/amber/src/main/scala/edu/uci/ics/texera/workflow/operators/udf/pythonV2/source/PythonUDFSourceOpDescV2.java

```diff
@@ -67,11 +68,11 @@ public class PythonUDFSourceOpDescV2 extends SourceOperatorDescriptor {
 
     @Override
     public OpExecConfig operatorExecutor(OperatorSchemaInfo operatorSchemaInfo) {
-        Function1<Object, IOperatorExecutor> exec = (i) ->
+        Function1<Object, IOperatorExecutor> exec = (Function1<Object, IOperatorExecutor> & Serializable) (i) ->
```

Comment:
can you elaborate on what this line is doing? I am not sure if I understand it.

Reply:
This is resolving the serialization error when scala is trying to serialize a java lambda function. I marked this lambda with `Serializable` so it can be correctly identified by scala.